adapters/ion-adapter/src/test/java/com/connexta/ion/replication/adapters/ion/IonNodeAdapterTest.java

```diff
@@ -73,18 +76,34 @@ public void createResource() {
     mockServer
         .expect(requestTo("http://localhost:1234/ingest"))
         .andExpect(method(HttpMethod.POST))
-        .andRespond(MockRestResponseCreators.withStatus(HttpStatus.ACCEPTED));
+        .andExpect(content().string(containsString(resourceFormData())))
+        .andExpect(content().string(containsString(correlationIdFormData())))
```

Comment:
Making assertions on MultiPartForm data turns out to be pretty difficult so comparing as string contents is the cleanest thing I could come up with. Better solutions are welcome